Website/src/items/renderings/edgewebsite/SessionsGrid.yml

```diff
@@ -18,7 +18,7 @@ SharedFields:
   Value: "{4D45DA3E-8128-422C-AFDA-E10672117E15}"
 - ID: "b5b27af1-25ef-405c-87ce-369b3a004016"
   Hint: Datasource Location
-  Value: ./Page Components|/sitecore/content/EdgeWebsite/Components/SessionsGrid
+  Value: 
```

Comment:
What is the reason why this Datasource location was removed?

Reply:
Good question - When I looked at this rendering, it was confusing because it was displaying the list of speakers as expected but I didn't see a datasource.

Looking at the rendering item, the "Rendering Contents Resolver" was set like the image below and is using the context item:

![rendering_resolver](https://user-images.githubusercontent.com/2945323/132673804-c1b9ecff-894b-4301-b5bc-89f10ac3ab82.jpg)

So decided to remove the datasource location since it does not seem to use a datasource (And the value would need to be updated to is kept to: ./Page Components|/sitecore/content/EdgeWebsite/**Data**/Components/SessionsGrid).

Reply:
That is a excellent reason. Good catch here!